src/antenna.c

```diff
@@ -105,31 +103,53 @@ void deploy_antenna(void) {
     }
 
```

Comment:
Should put a print at the beginning of `deploy_antenna()` saying that it is deploying now.

Reply:
Remove the TODO on line 64.